backend/station_manager/app/src/main/java/com/StationManager/app/storage/unitofwork/InMemoryUnitOfWork.java

```diff
@@ -0,0 +1,38 @@
+package com.StationManager.app.storage.unitofwork;
+
+import com.StationManager.app.storage.repository.*;
+import com.StationManager.app.storage.repository.inmemory.*;
+
+public class InMemoryUnitOfWork extends UnitOfWork {
+
+    private IClientRepository clientRepository;
+    private IHallRepository hallRepository;
+    private IPrivilegyRepository privilegyRepository;
+    private ITicketOfficeRepository ticketOfficeRepository;
+    private ITrainStationRepository trainStationRepository;
+
+    public InMemoryUnitOfWork() {
+        clientRepository = new InMemoryClientRepository();
```

Comment:
not sure about the differences, but you can also do that on the attribute level.
`public IHallRepository hallRepository = new InMemoryHallRepository();`

Reply:
I think there is no major differences. My idea was to have similar convention for all Unit of works (some, such as postgres may need initial parameters). Do you think it is better to put it as attribute level initialisation?